designer/server/src/common/helpers/auth/user-session.js

```diff
@@ -30,7 +30,7 @@ export function createUser(credentials, claims) {
   // Create user object (e.g. signed in token but no session)
   return /** @satisfies {UserCredentials} */ ({
     id: token.sub,
-    email: token.email ?? '',
+    email: token.email ?? token.unique_name ?? '',
```

Comment:
Noticed that the standard `email` claim isn't provided

This might be unique to `@defradev.onmicrosoft.com` but `unique_name` works